osconfig_tests/main.go

```diff
@@ -5,7 +5,6 @@
 //  You may obtain a copy of the License at
 //
 //      http://www.apache.org/licenses/LICENSE-2.0
-//
```

Comment:
why did you change this?

Reply:
this was not deliberate. will remove it.

Reply:
done